src/exm.gresource.xml.in

```diff
@@ -16,11 +16,16 @@
     <file>exm-info-bar.ui</file>
     <file>exm-info-bar-item.ui</file>
     <file>exm-screenshot-view.ui</file>
+    <file alias="gtk/help-overlay.ui">exm-help-overlay.ui</file>
     <file alias="suggestions.txt">res/suggestions.txt</file>
     <file preprocess="xml-stripblanks" alias="com.mattjakeman.ExtensionManager.metainfo.xml">../data/@appstream_file@</file>
     <file>style.css</file>
   </gresource>
 
+  <gresource prefix="/com/mattjakeman/ExtensionManager/Devel">
+    <file alias="gtk/help-overlay.ui">exm-help-overlay.ui</file>
+  </gresource>
```

Comment:
No longer dependent on #568, these are the lines we could skip with those changes.